python_modules/libraries/dagster-aws/dagster_aws/s3/io_manager.py

```diff
@@ -149,9 +149,13 @@ def handle_output(self, context: OutputContext, obj: Any) -> None:
         return self.inner_io_manager().handle_output(context, obj)
 
 
+# S3PickleIOManager used to be named ConfigurablePickledObjectS3IOManager, keep this symbol around for backcompat
+ConfigurablePickledObjectS3IOManager = S3PickleIOManager
```

Comment:
should we have a deprecation warning on `ConfigurablePickledObjectS3IOManager`? I'm not totally sure how to do that since it's just a symbol rename now

Reply:
I commented in other diff. Making it a subclass should do it